src/main/java/gift/repository/GiftJdbcRepository.java

```diff
@@ -93,15 +112,16 @@ public Gift modify(Long id, Gift gift) {
             params.add(gift.getGiftPhotoUrl());
         }
         if(params.isEmpty()){
```

Comment:
```suggestion
        if (params.isEmpty()) {
```

사소하지만, 코드 컨벤션을 지키기 위해 intellij의 Reformat Code 기능을 활용해보세요! 수시로 이용하시는 습관을 들이시면 좋습니다!
이후 협업을 하게 되었을 때, 이러한 컨벤션은 매우 중요합니다! (만약 누군가 code reformat 을 수행하지 않는다면 다른 누군가는 이 작업을 수행해야 합니다ㅠ)